code/obj/item/clothing/uniforms.dm

```diff
@@ -17,6 +17,7 @@
 	burn_possible = TRUE
 	health = 10
 	var/team_num
+	var/obj/item/cutting_product
```

Comment:
```suggestion
	var/cutting_product
```
Use doc comments rather than types to indicate the kind of type you mean, this looks like an instance var.

Reply:
Thanks for the input!!